### PR TITLE
Support git deploying DNX console apps as WebJobs

### PR DESCRIPTION
**Pull request by ahmelsayed**

### CODE REVIEW

**Comment by ahmelsayed** [comment]
This goes with this PR on KuduScript https://github.com/projectkudu/KuduScript/pull/31

File: Kudu.TestHarness/TestRepositories.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ internal class TestRepositories
             new TestRepositoryInfo("https://github.com/KuduApps/CustomDeploymentSettingsTest.git",  "bc0ce04"),
             new TestRepositoryInfo("https://github.com/KuduApps/CSharp6Web.git",                    "41d748f"),
             new TestRepositoryInfo("https://github.com/KuduApps/Dev11_Net45_Mvc4_WebAPI.git",       "bb9e6b4"),
+            new TestRepositoryInfo("https://github.com/KuduApps/DnxWebJobTest.git",                 "8450950"),
```
Comment (davidebbo) [inline Kudu.TestHarness/TestRepositories.cs]: That's a trigger job! :)

Comment (ahmelsayed) [inline Kudu.TestHarness/TestRepositories.cs]: We only really need it for the logs though, and it get's deleted right away.

Comment (davidebbo) [inline Kudu.TestHarness/TestRepositories.cs]: Ok, it's fine then.

**Comment by davidebbo** [comment]
:shipit:

**Pull request merged**